Gemfile.lock

```diff
@@ -652,10 +654,10 @@ GEM
       xmlmapper (~> 0.6)
     xmlmapper (0.7.2)
       nokogiri (~> 1.5)
-    xpath (2.0.0)
+    xpath (2.1.0)
       nokogiri (~> 1.3)
     zonebie (0.6.1)
-    zxcvbn-js (4.4.2)
+    zxcvbn-js (4.4.3)
```

Comment:
Looks like it's not available yet?
```
No compatible version found: zxcvbn@^4.4.3
```
Should I revert the gem until the node package is available?